### DISCUSSION TITLE
Move tab mute to the right! | Tab mute barely visible

### DISCUSSION THREAD

**Discussion by EkanaStone** [zen-browser/desktop#2418] [Ideas]
### What happened?

The button to mute tabs is barely visible as it disappears and is located directly on the favicon. I suggest that you shift the mute button to the right of the favicon and have it always showing when audio is playing. 

Here's an example of how I think it should look like:
![image](https://github.com/user-attachments/assets/bb686e29-ee86-4a80-a1b9-e105d072b6b1)

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.29

### Severity impact

Low

### What platform are you seeing the problem on?

Windows

**Comment by salamijason** [comment]
Will be moved to discussions as a feature request/suggestion. 

In the meantime, you can try using [this](https://zen-browser.app/mods/c5f7fb68-cc75-4df0-8b02-dc9ee13aa773) Zen Mod to enhance the visibility.
